### PR TITLE
magento/devdocs#: CLI. Create or edit an administrator. Examples

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds examples of usage `bin/magento admin:user:create` command.

## Affected DevDocs pages

-https://devdocs.magento.com/guides/v2.1/install-gde/install/cli/install-cli-subcommands-admin.html
-https://devdocs.magento.com/guides/v2.2/install-gde/install/cli/install-cli-subcommands-admin.html
-https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-subcommands-admin.html

## Links to Magento source code

- [AdminUserCreateCommand.php](https://github.com/magento/magento2/blob/2.3-develop/setup/src/Magento/Setup/Console/Command/AdminUserCreateCommand.php)

Thank you!

whatsnew
Added examples of [creating and updating a Magento admin user](https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-subcommands-admin.html) with the command line

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/install-gde/install/cli/install-cli-subcommands-admin.md [hunk 1]
```diff
@@ -43,24 +48,72 @@ Where the following table defines parameters and values:
...
+bin/magento admin:user:create
+```
+
+```terminal
+Admin user: John
+Admin password: 
+Admin email: j.doe.young@example.com
+Admin first name: John
+Admin last name: Doe Young
+```
```
Comment (ronak2ram) [inline guides/v2.1/install-gde/install/cli/install-cli-subcommands-admin.md]: Hi @atwixfirster,
When we run command `bin/magento admin:user:create` then we get this line no 68 output for Magento 2.3.x version and 2.2.x. so can you please revert this changes from doc file v2.1 and create a file for v2.2 and v2.3 doc with above change.

Comment (atwixfirster) [reply guides/v2.1/install-gde/install/cli/install-cli-subcommands-admin.md]: **done**

Thank you for notice, @ronak2ram ! 👍 

Could you please review now?

Comment (ronak2ram) [reply guides/v2.1/install-gde/install/cli/install-cli-subcommands-admin.md]: Hi @atwixfirster 
thanks for the update.

**Comment by jeff-matthews** [comment]
running tests

**Comment by jeff-matthews** [comment]
running tests
